tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -218,32 +218,17 @@ export class Ide {
         await this.waitStatusBarContains(expectedTextInStatusBar, 20000);
     }
 
-    async closeAllNotifications() {
-        const notificationLocator: By = By.css('.theia-Notification');
-
+    async closeAllNotifications(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
```

Comment:
WDYT about this variant?
```
    async closeAllNotifications(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
        Logger.debug('Ide.closeAllNotifications');

        await this.clickStatusBarNotifications(timeout);
        await this.clickNotificationCenterCloseAllButton(timeout);
        await this.waitNotificationCenterClosedAll(timeout);
        await this.waitNotificationCenterDisappearance(timeout);
    }

    async clickStatusBarNotifications(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
        Logger.debug('Ide.clickStatusBarNotifications');

        const statusBarNotificationsLocator: By = By.xpath(`//div[@id="theia-statusBar"]/div[@class="area right"]//div[contains(@title, 'Notification')]`);
        await this.driverHelper.waitAndClick(statusBarNotificationsLocator, timeout);
    }

    async clickNotificationCenterCloseAllButton(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
        Logger.debug('Ide.clickNotificationCenterCloseAllButton');

        const notificationCenterActionCloseAllButtonLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center open')]//ul[@class="theia-notification-actions"]/li[@class="clear-all"]`);
        await this.driverHelper.waitAndClick(notificationCenterActionCloseAllButtonLocator, timeout);
    }

    async waitNotificationCenterClosedAll(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
        Logger.debug('Ide.waitNotificationCenterClosedAll');

        const notificationCenterClosedLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center closed')]`);
        await this.driverHelper.waitPresence(notificationCenterClosedLocator, timeout);
    }

    async waitNotificationCenterDisappearance(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
        Logger.debug('Ide.waitNotificationCenterDisappearance');

        const notificationCenterContainerNotificationsLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center')]//div[@class='theia-notification-list']//*`);
        await this.driverHelper.waitDisappearance(notificationCenterContainerNotificationsLocator, timeout);
    }
```
Besides when we should specify a place where intracted element is placed (NotificationCenter <--- CloseButton)
it is clear signal for creation new pageobject (NotificationCenter).

Reply:
Hello Igor.
I'm happy to separate it into dedicated methods.

Regarding NotificationCenter, there already is an issue opened to create a pageobject for that and for notifications themselves https://github.com/eclipse/che/issues/15700 . I wouldn't include that into this PR and instead open it as a completely separate PR.

Reply:
Yes, a good idea, thank you for the answer.